.cicd/defaults.json

```diff
@@ -0,0 +1,10 @@
+{
+   "leap-dev":{
+      "target":"*",
+      "prerelease":true
+   },
+   "cdt":{
+      "target":"*",
+      "prerelease":true
+   }
+}
```

Comment:
I would not use pre-releases for CICD validation. They are, by definition, alpha or beta software. If there are bugs in pre-releases from other repos/systems, and there likely will be, then we shouldn't fail the CICD for this repo.

If the goal is to test the contracts with release candidates, we should come up with a different strategy for that.

Reply:
The intended behavior may (or may not?) be different depending on if it's a release branch or the main trunk. This PR is going to a release branch, and when doing the build now
```
CMake Warning at tests/CMakeLists.txt:16 (message):
  Using Leap version 3.2.0-rc2 even though it exceeds the maximum supported
  version of 3.1; continuing with configuration, however build may fail.

  It is recommended to use Leap version 3.1.x
```
So I wonder what the process if for coming along and changing this branch's config in cmake to whitelist 3.2
https://github.com/eosnetworkfoundation/eos-system-contracts/blob/4d8a5f7bb89587b2751dea9ccc6eb41f1849beac/tests/CMakeLists.txt#L3-L4

And then when making that change would we change it to
```json
   "leap-dev":{
      "target":"3.1 || 3.2",
      "prerelease":false
   },
```

To just get this merged, at this point, if switching over to `"prerelease":false` is all it takes then I'm certainly fine with that. I think someone may need to come along in the future and adjust these knobs some more to their liking.

Reply:
This reads to me like there is an issue with the CMake, rather than there being a legitimate reason to use non-stable software for the CICD system. Is that accurate?

Reply:
I decided to change the defaults in this branch to be 3.x, non-prereleases. Then when we merge to main I'll switch it to something else. Someone else who has a better feel can change it later if needed.